### PR TITLE
split NoTils based on language

### PR DESCRIPTION
**Pull request by blundell**
- now have Java & Android modules (and releases)
 - removes crazy provided android jar dependency
 - setting version to 3.0.0 for all releases
 - Android depends on Java and we must always release together

Paired with @ouchadam

### CODE REVIEW

**Comment by blundell** [comment]
superceeds #67

**Comment by ouchadam** [comment]
LGTM :+1:

File: android/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.novoda.android">
+
+    <application android:allowBackup="true"
+        android:label="@string/app_name"
+        android:supportsRtl="true"
```
Comment (amlcurran) [inline android/src/main/AndroidManifest.xml]: will this get merged into releases of apps that depend on this library? Don't want to force all our apps to support RTL

Comment (blundell) [inline android/src/main/AndroidManifest.xml]: good question. no any downstream manifest has priority so they can set it to false

Comment (amlcurran) [inline android/src/main/AndroidManifest.xml]: If they have to set it as false then make sure to update the usage
documentation

On Tuesday, 1 March 2016, Paul Blundell <notifications@github.com> wrote:

> In android/src/main/AndroidManifest.xml
> <https://github.com/novoda/notils/pull/68#discussion_r54548691>:
>
> > @@ -0,0 +1,11 @@
> > +<manifest xmlns:android="http://schemas.android.com/apk/res/android"
> > +    package="com.novoda.android">
> > +
> > +    <application android:allowBackup="true"
> > +        android:label="@string/app_name"
> > +        android:supportsRtl="true"
>
> good question. no any downstream manifest has priority so they can set it
> to false
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/novoda/notils/pull/68/files#r54548691>.
>

-- 
Disclaimer: The information in this e-mail and any attachments is the 
property of Novoda Ltd and is confidential and may be legally privileged. 
This e-mail is intended solely for the person or organisation to which it 
is addressed. Any disclosure, copying or other use of the information by 
any person or organisation who is not the intended recipient is strictly 
prohibited and may be unlawful. If you have received this e-mail in error, 
please inform the sender immediately and delete/destroy this e-mail and any 
copies of it. Novoda Ltd has taken reasonable precautions to minimise the 
risk of any software viruses which may damage your systems, but we advise 
that you take the necessary steps to ensure that no virus contamination is 
suffered. Novoda Ltd does not accept any liability for any loss or damage 
caused by the transmission of any virus.

Novoda Ltd, Company No: 347444, Registered in Scotland Registered 
Office: C/O Alexander Sloan, 38 Cadogan Street, Glasgow, G2 7HF, Scotland.  VAT 
Registration Number  GB 984 2525 93

Comment (blundell) [inline android/src/main/AndroidManifest.xml]: what do you mean sorry?

Comment (amlcurran) [inline android/src/main/AndroidManifest.xml]: as in, if a downstream person now _has_ to set it false (and it isn't false by default) that should be mentioned in the release notes when we come to release it :)

Comment (blundell) [inline android/src/main/AndroidManifest.xml]: if they have it set to false they will override this and `false` with be the RTL. If they don't have it set then the default is true like this?

Comment (blundell) [inline android/src/main/AndroidManifest.xml]: but I was thinking if our library supprts RTL (which I believe it does) we should have it set to `true`. 

We are declaring our library supports RTL

**Pull request merged**